src/openrct2/ride/Ride.h

```diff
@@ -375,6 +375,7 @@ struct Ride
     uint8_t num_circuits;
     CoordsXYZ CableLiftLoc;
     uint16_t cable_lift;
+    uint8_t selling_item_color_is_random;
```

Comment:
we use british english in this code base and also new fields should be CamelCase
`SellingItemColourIsRandom`